csharp/extractor/Semmle.Extraction.CSharp.Standalone/Extractor.cs

```diff
@@ -60,10 +60,9 @@ private static IEnumerable<Action> GetResolvedReferencesStandalone(IEnumerable<s
             {
                 try
                 {
-                    FileUtils.TryDelete(output.FullName);
                     if (shouldCleanUpContainingFolder)
                     {
-                        output.Directory?.Delete(true);
+                        FileUtils.TryDelete(output.FullName);
```

Comment:
This is a small bugfix, which only has an impact on the case when the temp directory is not in the DB scratch folder. So, it only impacts running/debugging the Standalone extractor directly.